modules/accessing-inference-endpoint-for-model-deployed-on-single-model-serving-platform.adoc

```diff
@@ -31,6 +31,36 @@ The inference endpoint for the model is shown in the *Inference endpoint* field.
 // * `:443/api/v1/task/text-classification`
 // * `:443/api/v1/task/token-classification`
 
+*Caikit Standalone ServingRuntime*
+
+.HTTP endpoints
+* `/api/v1/task/embedding`
+* `/api/v1/task/embedding-tasks`
+* `/api/v1/task/sentence-similarity`
+* `/api/v1/task/sentence-similarity-tasks`
+* `/api/v1/task/rerank`
+* `/api/v1/task/rerank-tasks`
+
+.gRPC endpoints
+* `:443 caikit.runtime.Nlp.NlpService/EmbeddingTaskPredict`
+* `:443 caikit.runtime.Nlp.NlpService/EmbeddingTasksPredict`
+* `:443 caikit.runtime.Nlp.NlpService/SentenceSimilarityTaskPredict`
+* `:443 caikit.runtime.Nlp.NlpService/SentenceSimilarityTasksPredict`
+* `:443 caikit.runtime.Nlp.NlpService/RerankTaskPredict`
+* `:443 caikit.runtime.Nlp.NlpService/RerankTasksPredict`
++
+ifdef::upstream[]
+NOTE: By default, the Caikit Standalone ServingRuntime exposes REST endpoints for use. To use gRPC protocol, manually deploy a custom Caikit Standalone ServingRuntime. For more information, see link:{odhdocshome}/serving-models/#adding-a-custom-model-serving-runtime-for-the-single-model-serving-platform_serving-large-models[Adding a custom model-serving runtime for the single-model serving platform]. 
```

Comment:
We should probably settle on 'HTTP endpoints', rather than 'REST endpoints'. I think HTTP will be the protocol that's indicated next to the inference endpoint in the dashboard.

Reply:
It's displayed as "REST" but I'm fine with either:
<img width="1728" alt="Screenshot 2024-07-02 at 9 46 30 AM" src="https://github.com/opendatahub-io/opendatahub-documentation/assets/77992688/b010a8ad-648e-4c27-9c19-12b920e761b3">

Reply:
Ah, thanks for confirming, @christinaexyou :+1: In that case, let's do the opposite and title the bullet list of endpoints to match.